content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md

```diff
@@ -6,6 +6,27 @@ menu:
     parent: About the project
 ---
 
+
+## v1.5.8 [2020-1-6]
+
+## Breaking changes
+
+- Remove support for darwin/386 builds (Go no longer supports), and add support for darwin/arm64 builds.
+
+## Features
+
+- Add InfluxDB `subscription-path` configuration option to allow Kapacitor to run behind a reverse proxy, thanks @aspring!
+- Add support for HTTP sources in `SideloadNode` configuration, thanks @jregovic!
+- Add support for correlate in the Alerta AlertNode, thanks @nermolaev!
```

Comment:
Is correlate a new event handler option? configuration option? and/or option to include in task template scripts? Do we have an example to show how to use correlate?

Reply:
Yes

Reply:
I do not have any examples, sorry, the bonito folks will know more.

Reply:
Thank you, @docmerlin, I don't see "correlate" as an option in the alerta section of the Kapacitor config. Should that be there?